src/main/java/by/issoft/mandrik/pages/AbstractPage.java

```diff
@@ -0,0 +1,14 @@
+package by.issoft.mandrik.pages;
+
+import org.openqa.selenium.WebDriver;
+
+public abstract class AbstractPage {
+    WebDriver driver;
+
+    public abstract void openPage();
+
+    AbstractPage(WebDriver driver)
+    {
```

Comment:
Incorrect formatting

Reply:
You have class, which provides access to WebDriver instance. Why do you still pass it as parameter?